src/foam/util/Emails/EmailsUtility.java

```diff
@@ -137,7 +137,8 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
     try {
       cts.apply(userX, group, emailMessage, templateArgs);
     } catch (Exception e) {
-      logger.error(e);
+
+      logger.error("Problem with template: " + templateName, e);
```

Comment:
Please also add an alarm here - warning is fine. 
name: EmailTemplate
note: add the templateName